packages/netlify-cms-backend-github/src/API.ts

```diff
@@ -870,145 +870,103 @@ export default class API {
         entry: { path: entry.path, sha: entry.sha },
         files: mediaFilesList,
       };
-      const updatedMetadata = { ...metadata, pr, title, description, objects };
 
-      if (options.hasAssetStore) {
```

Comment:
Only reference I found regarding this code is the comment here:
https://github.com/netlify/netlify-cms/commit/edf0a3afdc01690c9075476354db7a6b46e84f16#diff-b4ba42c688a23ed9cec5139daec2c157L437

Not sure why rebasing wasn't happening when using asset store as it is currently only used for uploading media files.
Entries are still committed to the repository.

Reply:
It's because rebasing was purely about media files being available in deploy previews - since they were going straight to master, the branch needed to be rebased so new media files were deployed. If an asset store was in use, there was no need to rebase. Cool to remove 👍